### PR TITLE
[perf] upgrade realm and improve schemas/queries

### PR DESCRIPTION
**Pull request by yenda**
* upgrade to realm 2.28 to benefit from perf improvements
* remove user-statuses and replace by seen and outgoing-status fields
to get rid of a lot of bloat queries and computations and in-queries which don't
work in this version of realm
* remove unused seen message, bottom-infos

## Tests
Tested on Android

Tested:

* send and receive messages from public and private chats

To test:

* start from nightlies with some chats that have unread messages and unsent messages
* install this version and check that the migrations worked

MV5BZjkyYThhNWMtYWRiZi00ZjQyLThiYzYtZjkwYmVkYTRlYTIwXkEyXkFqcGdeQXVyNTk1ODg5ODA@ _V1_

status: wip

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]

**Comment by jakubgs** [comment]
I wonder if this might help with #8326 .

**Review by jakubgs — APPROVED** [review]
I'm approving only the `yarn.lock` changed, the Clojure stuff is black magic.

File: src/status_im/chat/models/message.cljs [hunk 1]
```diff
@@ -39,9 +38,9 @@
   [{:keys [content content-type] :as message} chat-id current-chat?]
   (let [emoji? (message-content/emoji-only-content? content)]
     ;; TODO janherich: enable the animations again once we can do them more efficiently
-    (cond-> (assoc message :appearing? true)
-      (not current-chat?)
-      (assoc :appearing? false)
+    (cond-> message
+      current-chat?
+      (assoc :seen true)
```
Comment (flexsurfer) [inline src/status_im/chat/models/message.cljs:43]: could you elaborate?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/chat/models/message.cljs (hunk 1 above)
Comment (yenda) [reply src/status_im/chat/models/message.cljs:43]: `appearing` is useless, not needed.
If the message is received in the current chat it is considered seen. It was already the case before just not done here

**Comment by asemiankevich** [comment]
@yenda is it something that was broken here? 
image

**Comment by yenda** [comment]
@asemiankevich this is a bug I've seen a few times already in other PR, I have never been able to reproduce in the emulator so I don't really now where it is coming from

**Comment by asemiankevich** [comment]
> @asemiankevich this is a bug I've seen a few times already in other PR, I have never been able to reproduce in the emulator so I don't really now where it is coming from

@yenda i've logged it separately, no worries! Apart from this current PR looks good to me, upgrade looks okay-ish.

**Pull request merged**